### PR TITLE
Update markup for newsletter checkbox

### PR DESCRIPTION
**Pull request by cdrini**
Fix; got a request to update this markup from Jenica to be consistent.

### Testing
Tested English/Spanish both work

### Stakeholders
@jbuckner

### CODE REVIEW

File: openlibrary/i18n/messages.pot [hunk 1]
```diff
@@ -135,9 +135,7 @@ msgstr ""
 
 #: forms.py:62
 msgid ""
-"Send me a monthly newsletter from the <a "
-"href=\"https://archive.org/\">Internet Archive</a>, the non-profit that "
-"runs Open Library"
+"I want to receive news, announcements, and resources from the <a href="https://archive.org/">Internet Archive</a>, the non-profit that runs Open Library."
```
Comment (cdrini) [inline openlibrary/i18n/messages.pot:138]:
```suggestion
"I want to receive news, announcements, and resources from the <a href=\"https://archive.org/\">Internet Archive</a>, the non-profit that runs Open Library."
```

File: openlibrary/i18n/fr/messages.po [hunk 2]
```diff
@@ -143,9 +143,7 @@ msgstr "Les mots de passe ne correspondent pas."
 
 #: forms.py:62
 msgid ""
-"Send me a monthly newsletter from the <a "
-"href=\"https://archive.org/\">Internet Archive</a>, the non-profit that "
-"runs Open Library"
+"I want to receive news, announcements, and resources from the <a href="https://archive.org/">Internet Archive</a>, the non-profit that runs Open Library."
```
Comment (cdrini) [inline openlibrary/i18n/fr/messages.po:146]:
```suggestion
"I want to receive news, announcements, and resources from the <a href=\"https://archive.org/\">Internet Archive</a>, the non-profit that runs Open Library."
```

File: openlibrary/i18n/es/messages.po [hunk 3]
```diff
@@ -128,8 +128,8 @@ msgid "Passwords didn't match."
 msgstr "Las contraseñas no coinciden."
 
 #: forms.py:62
-msgid "Send me a monthly newsletter from the <a href=\"https://archive.org/\">Internet Archive</a>, the non-profit that runs Open Library"
-msgstr "Envíame un boletín mensual de <a href=\"https://archive.org/\">Internet Archive</a>, la organización sin fines de lucro que administra Open Library"
+msgid "I want to receive news, announcements, and resources from the <a href="https://archive.org/">Internet Archive</a>, the non-profit that runs Open Library."
```
Comment (cdrini) [inline openlibrary/i18n/es/messages.po:131]:
```suggestion
msgid "I want to receive news, announcements, and resources from the <a href=\"https://archive.org/\">Internet Archive</a>, the non-profit that runs Open Library."
```

**Comment by jbuckner** [comment]
Looks good 👍

File: openlibrary/i18n/es/messages.po [hunk 4]
```diff
@@ -128,8 +128,8 @@ msgid "Passwords didn't match."
 msgstr "Las contraseñas no coinciden."
 
 #: forms.py:62
-msgid "Send me a monthly newsletter from the <a href=\"https://archive.org/\">Internet Archive</a>, the non-profit that runs Open Library"
-msgstr "Envíame un boletín mensual de <a href=\"https://archive.org/\">Internet Archive</a>, la organización sin fines de lucro que administra Open Library"
+msgid "I want to receive news, announcements, and resources from the <a href=\"https://archive.org/\">Internet Archive</a>, the non-profit that runs Open Library."
+msgstr "Envíame un boletín mensual de <a href=\"https://archive.org/\">Internet Archive</a>, la organización sin fines de lucro que administra Open Library."
```
Comment (jbuckner) [inline openlibrary/i18n/es/messages.po:132]: We might need to update the translation as well.

Comment (cdrini) [reply openlibrary/i18n/es/messages.po:132]: 👍 @dcapillae Any chance you could update this? If you leave a suggestion on GitHub, I can include it as part of this PR :)

Comment (dcapillae) [reply openlibrary/i18n/es/messages.po:132]:
```suggestion
msgstr "Quiero recibir noticias, anuncios y recursos de <a href=\"https://archive.org/\">Internet Archive</a>, la organización sin fines de lucro que administra Open Library."
```

Comment (cdrini) [reply openlibrary/i18n/es/messages.po]: Muchas gracias @dcapillae !

**Comment by cdrini** [comment]
Auto merging since this is a markup change

**Pull request merged**